apps/stress-test/scripts/utils/configureYargs.js

```diff
@@ -0,0 +1,119 @@
+const { getBrowsers } = require('../../dist/getBrowsers');
```

Comment:
When running `yarn && yarn buildto @fluentui/stress-test` and then the steps you mentioned in the description, I get an error saying that module `getBrowsers` is not found. I looked in the directory `../../dist` and it does seem to be missing. Should this be generated somewhere or before running `configureYargs` inside `run`?

Reply:
Here's a screenshot of the error:
<img width="623" alt="image" src="https://user-images.githubusercontent.com/5953191/186999296-7f17cbed-301d-46fe-9ed6-ce0daa885400.png">

Reply:
Strange...looks like the file in question was lost when I rebased. Just pushed an update. Can you try again?

Reply:
It works!